README.md

```diff
@@ -117,6 +117,47 @@ This profile creates two SSL certificates.
 # teardown example
 ./dsp-k8s-deploy/application-teardown.sh -j broad-wb-perf2.json -p ssl
 ```
+### `sam-sa`
+
+This profile creates:
+
+1. A sam service account for use by Sam instances
+
+This profile must be applied with `-a`, which uses your local application default
+credentials instead of a terraform service account. To generate your own ADC, run
+`gcloud auth application-default login` and follow the instructions.
```

Comment:
why does this require ADC? In my case, using my account, it all works without ADC and in fact fails when I try to use ADC.

Reply:
Good catch, I misunderstood how this environment was configured. SAs can only be created by an identity (SA or user) with Owner permissions on the project. You don't have Owner on these projects (yet, I can add you).

When we've used terraform in the past, we have not given the terraform SA the Owner permission because that seemed scary. But I notice that the terraform SA used in these projects does have that. I'd prefer to knock that down to Editor (what we use for tf in prod currently) and require personal ADCs to apply IAM roles, but I'd say that's a question for Secops to rule on. For now the README is incorrect.

Reply:
thanks - I've updated the README!